app/questionnaire/router.py

```diff
@@ -64,10 +69,9 @@ def get_next_location_url(self, location, routing_path):
         ):
             return last_block_location.url()
 
-        if self.is_survey_complete():
+        if self.is_survey_complete and not hub_enabled:
```

Comment:
Unless I'm missing something, `self.is_survey_complete` isn't being invoked and isn't a property either?